### ISSUE TITLE
Avoid running integration tests for pre-release PRs

### ISSUE THREAD

**Issue by Strift** [meilisearch/integration-guides#309]
## **Description**

The GitHub integration currently includes two workflows:

- Integration Tests: These run tests against the latest stable release (`latest` Docker tag)
- Pre-release Integration Tests: These run tests against the latest version of Meilisearch (e.g., `meilisearch:v1.x.y.rc-z` Docker tag)

The Integration Tests often fail on `bump-*` branches because these branches include updates that depend on the target version (the new release), which isn’t yet compatible with the stable version used in these tests.

## **Proposed Solution**

Update the CI configuration to ensure that only Pre-release Integration Tests are executed for bump-* branches. This adjustment will align the tests with the appropriate Meilisearch version and prevent unnecessary failures.

**Comment by brunoocasali** [comment]
I agree with @curquiza, @Strift her answer was self-explanatory, I don't see what I can add to the discussion 😅 
I will close this issue since I see your thumbs-up reaction in her answer as well :D
